span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanToLogRecordsTransformer.java

```diff
@@ -23,6 +28,12 @@
   private static final Logger LOG =
       LoggerFactory.getLogger(JaegerSpanToLogRecordsTransformer.class);
 
+  private static final String VALID_SPAN_WITH_LOGS_RECEIVED_COUNT =
```

Comment:
Let's add both the metrics here as well - https://github.com/hypertrace/hypertrace/tree/main/kubernetes/monitoring#hypetrace-ingestion-layer

Reply:
I'll add